examples/with-webpack-bundle-analyzer/next.config.js

```diff
@@ -0,0 +1,17 @@
+const {BundleAnalyzerPlugin} = require('webpack-bundle-analyzer')
+module.exports = {
+  webpack: (config, { dev }) => {
+    // Perform customizations to config
+    config.plugins.push(
```

Comment:
shouldn't the bundle analyzer be loaded only in `dev` (and dep moved to devDependencies)?